src/expressions/visitors/TimeIndexVisitor.cpp

```diff
@@ -89,7 +89,7 @@ TimeIndex TimeIndexVisitor::visit(const Nodes::NegationNode* neg)
     return dispatch(neg->child());
 }
 
-TimeIndex TimeIndexVisitor::visit(const Nodes::PortFieldNode* port_field_node)
+TimeIndex TimeIndexVisitor::visit([[maybe_unused]] const Nodes::PortFieldNode* port_field_node)
```

Comment:
better use [unnamed parameter ](https://github.com/isocpp/CppCoreGuidelines/issues/808#issuecomment-263981643)
```suggestion
TimeIndex TimeIndexVisitor::visit( const Nodes::PortFieldNode* )
```